lib/utils.js

```diff
@@ -0,0 +1,3 @@
+export function areObjectsEmpty(a, b) {
+  return typeof data === 'object' && typeof previousData === 'object' && Object.values(a).length === 0 && Object.values(b).length === 0
```

Comment:
Where do `data` and `previous` data come from and what is this method achieving exactly?

Reply:
You are right, this was a copy and paste mistake from my other PR, the correct values are `a` and `b`